### DISCUSSION TITLE
有关“用户文件的信息说明”中的限制

### DISCUSSION THREAD

**Discussion by ghost** [liusy58/NJU_AdvancedProgramming_SP22#109] [Q&A] [answered]
请问这些限制是对于测试数据的限制，还是说，要求程序限制这些输入满足要求，不满足要求就报错？  
例如
![image](https://user-images.githubusercontent.com/100082486/157015301-1a5f7f3c-42c6-434e-98ee-551b0ea5c9b2.png)
，测试数据中是否会有超过此限制的用户名，然后要求程序必须针对超过10字符限制的用户名报错？

**Comment by binbin00** [comment, marked as answer]
肯定会，ppt里有强调：任何用户输入的地方都有可能出现各种问题。

**Reply by VansamaZWT** [reply]
那请问我是必须要报错出来还是在代码运行中做相应处理使程序正常运行即可
